features/edit_preferences.feature

```diff
@@ -9,9 +9,9 @@ Background:
 	Given I am logged in with email "john.apple@gmail.com" and password "hello1"
   	And I am on the edit preferences page
 
-Scenario: Input all data and preferences
+#Scenario: Input all data and preferences
```

Comment:
These incomplete scenarios were causing Travis CI builds to fail. Will need to fill these in

Reply:
Commented out for now - will include as part of next iterations work!